### PR TITLE
ref(toolbar): rename view files, classes, and login path

### PR DESCRIPTION
**Pull request by aliu39**
Renaming to match the conventions used by other views/endpoints, for ex sentry.replays.endpoints. 

The only thing this affects for our other code is the login URL. IMO the login path should be /login instead of /login-success, to make the purpose of the view more clear. login-success.html (the pop-up) is only returned on success.

### CODE REVIEW

**Comment by ryan953** [comment]
> /login instead of /login-success, to make the purpose of the view more clear. login-success.html (the pop-up) is only returned on success.

I'm not following that train of though, the url `/toolbar/:org/:project/login-success/` does not preform a login, it's doesn't accept username/password as it's a GET only route. It would be more appropriate to relate or associate it with a route like `/home` or `/welcome` or something like that. 

Either way, we shouldn't be changing URLs, there is code that depends on these urls. When the user clicks 'login' from the toolbar we want to send them to the "homepage" aka the "you are logged in now" aka the "login is successful" page. I think the urls are fine for now, if we're going to change the contract we'll need a strong reason.

**Comment by aliu39** [comment]
> > /login instead of /login-success, to make the purpose of the view more clear. login-success.html (the pop-up) is only returned on success.
> 
> I'm not following that train of though, the url `/toolbar/:org/:project/login-success/` does not preform a login, it's doesn't accept username/password as it's a GET only route. It would be more appropriate to relate or associate it with a route like `/home` or `/welcome` or something like that.
> 
> Either way, we shouldn't be changing URLs, there is code that depends on these urls. When the user clicks 'login' from the toolbar we want to send them to the "homepage" aka the "you are logged in now" aka the "login is successful" page. I think the urls are fine for now, if we're going to change the contract we'll need a strong reason.

Ok, revisiting this, this makes sense. What I was thinking before, is that for our specific auth flow, I login by requesting this resource. But you're right it doesn't make sense to GET on a login endpoint. The login happens from the redirect, and if we're already authenticated, we skip it. It could also confuse users into thinking there's a separate login just for toolbar.

Tldr; yes let's keep /login-success 👌
